common/src/main/java/com/alibaba/nacos/common/notify/Event.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.common.notify;
+
+import java.io.Serializable;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * An abstract class for event.
+ *
+ * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
+ * @author zongtanghu
+ */
+@SuppressWarnings({"PMD.AbstractClassShouldStartWithAbstractNamingRule", "PMD.ConstantFieldShouldBeUpperCaseRule"})
+public abstract class Event implements Serializable {
+    
+    private static final AtomicLong sequence = new AtomicLong(0);
```

Comment:
I mean that  `no-->sequence`  the constant still is SEQUENCE.

Reply:
Okay, sorry , I disunderstand what's your meaning.

Reply:
I will fix this in next commit.